components/Card.jsx

```diff
@@ -11,23 +11,52 @@ export default function Card({ dataset, location }) {
 
   const cardData = dataset.cardData;
   return (
-    <div className="border-2">
-      {<h2 className="capitalize">{dictionary.hasOwnProperty(dataset.indicator) ? (<><span tabIndex={0} onMouseEnter={handleHover} onMouseLeave={handleHover} onFocus={handleHover} onBlur={handleHover}>{dataset.indicator}</span>{" "}<span className={`${hover ? "visible" : "invisible"}`}>{dictionary[dataset.indicator]}</span></>) : dataset.indicator}</h2>}
-      <h3>{dataset.cardData.locationData.Value}</h3>
-      <Link href={`/${location}/indicator/${dataset.indicator}`}>
-        <a>
-          <ul>
-            <li>This ranks {cardData.ranking}/33 of the London Boroughs</li>
-            <li>The London value is {cardData.londonData}</li>
-            <li>The Uk value is {cardData.ukData}</li>
-            {cardData.change > 0 ? (
-              <li>This has increased by {cardData.change.toPrecision(3)}%</li>
+    <div className="flex flex-col w-1/4 justify-evenly">
+      <div className="bg-ecnmy-white mb-1 flex flex-wrap flex-col rounded-t-lg">
+        {
+          <h2 className="capitalize bg-ecnmy-grape text-ecnmy-white m-3 p-2 self-center text-center rounded-lg w-10/12 text-lg font-bold">
```

Comment:
`self-center ` only works with `grid` or `flex` i think not sure if its doing anything

Reply:
Self center still needed to center the purple box